app/views/admin/CodeLogsView.coffee

```diff
@@ -0,0 +1,49 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/admin/codelogs-view'
+CodeLogCollection = require 'collections/CodeLogs'
+CodeLog = require 'models/CodeLog'
+utils = require 'core/utils'
+
+
+module.exports = class CodeLogsView extends RootView
+  template: template
+  id: 'codelogs-view'
+  tooltip: null
+  events:
+    'click .playback': 'onClickPlayback'
+
+  constructor: (options) ->
+    super options
+    @spade = new Spade()
+    @codelogs = new CodeLogCollection()
+    @listenTo(@codelogs, 'sync', @onThangsLoaded)
+    @supermodel.loadCollection(@codelogs, 'codelogs')
```

Comment:
Use `initialize` instead of `constructor`.

I think listening to `onThangsLoaded` should be unnecessary. If the supermodel is tracking requests, it'll render the whole view once everything has successfully loaded.

We're moving away from `loadCollection` and alike (so many things in flux). Instead, use `@supermodel.trackRequest(@codelogs.fetch())`. That way you're triggering the collection request directly, rather than indirectly through the supermodel.